### PR TITLE
docs: Alex Russell Approved javascript framework

### PR DESCRIPTION
**Pull request by PatrickJS**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

adding the fact that Qwik is an Alex Russell Approved javascript framework

### CODE REVIEW

**Review by gioboa — APPROVED** [review]

File: packages/docs/src/routes/docs/(qwik)/faq/index.mdx [hunk 1]
```diff
@@ -269,6 +270,10 @@ Most of Qwik is written in TypeScript, a superset of JavaScript that adds option
 
 Yes! there is a growing community of Qwik developers at [Discord](https://qwik.builder.io/chat) and [GitHub](https://github.com/BuilderIO/qwik). They are making amazing contributions to the framework, building sites at scale, and helping each other. [Join us](https://qwik.builder.io/chat).
 
+## Is Qwik an "Alex Russell Approved" framework?
+
+Yes! Alex Russell ([@slightlylate](https://twitter.com/slightlylate)), known for his contributions to PWAs, W3C TAG, WC, TC39 & ES6, Chrome Frame, Dojo, and more, is often highly critical of JavaScript frameworks. Nonetheless, he approves of Qwik.
+
```
Comment (gioboa) [inline packages/docs/src/routes/docs/(qwik)/faq/index.mdx:276]: Can you mention a specific post about this please?

**Comment by maiieul** [comment]
This is a fun one :sweat_smile:

File: packages/docs/src/routes/docs/(qwik)/faq/index.mdx (hunk 1 above)
Comment (NikitaPuzyryov) [reply packages/docs/src/routes/docs/(qwik)/faq/index.mdx:276]: Uhm, how come this is "resolved"?

Comment (PatrickJS) [reply packages/docs/src/routes/docs/(qwik)/faq/index.mdx:276]: @mhevery asked him directly

Comment (NikitaPuzyryov) [reply packages/docs/src/routes/docs/(qwik)/faq/index.mdx:276]: For posterity: citation added in #5381

**Pull request merged**